instrumentation/restclient/lib/opentelemetry/instrumentation/restclient/patches/request.rb

```diff
@@ -21,7 +21,7 @@ def execute(&block)
           def trace_request # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
             http_method = method.upcase
             instrumentation_attrs = {
-              'http.method' => http_method,
+              'http.method' => http_method.to_s,
```

Comment:
@james-o-johnstone Thank you for this contribution! 

May I ask that you also include a unit tests accompanying this change to ensure we avoid regressions?

Reply:
Hey @arielvalentin, sure, good idea. I have added a unit test.

Reply:
Thank you!